prombench/design.svg

```diff
@@ -1,3 +1,3 @@
pointer-events="none"/><g transform="translate(677.5,331.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="117" height="16" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 15px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 118px; white-space: nowrap; overflow-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;white-space:normal;">Fake WebServers</div></div></foreignObject><text x="59" y="16" fill="#000000" text-anchor="middle" font-size="15px" font-family="Helvetica">Fake WebServers</text></switch></g><path d="M 257 187 L 508 187 L 508 260.63" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 508 265.88 L 504.5 258.88 L 508 260.63 L 511.5 258.88 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><rect x="135" y="157" width="122" height="60" rx="9" ry="9" fill="#fbebff" stroke="#000099" pointer-events="none"/><g transform="translate(152.5,173.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="87" height="27" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 88px; white-space: nowrap; overflow-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;white-space:normal;"><div style="text-align: left"><span style="color: rgb(51 , 51 , 51) ; text-align: center">GitHub Actions </span></div><div style="text-align: left"><span style="color: rgb(51 , 51 , 51) ; text-align: center">starts a new test</span><br /></div></div></div></foreignObject><text x="44" y="20" fill="#000000" text-anchor="middle" font-size="12px" font-family="Helvetica">[Not supported by viewer]</text></switch></g><rect x="155" y="479.5" width="160" height="50" rx="7.5" ry="7.5" fill="#dae8fc" stroke="#6c8ebf" pointer-events="none"/><g transform="translate(159.5,487.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="151" height="34" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 15px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 152px; white-space: nowrap; overflow-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;white-space:normal;"><font>Loki<br /><font style="font-size: 10px">(promtail will push metrics to Loki)</font></font></div></div></foreignObject><text x=
```

Comment:
the line wrapping is not right on some blocks
https://github.com/prometheus/prombench/tree/d6783184d6e09887d261ad6ece5481b1b3a2fbfe/prombench

Reply:
okay will update this